testsuite/MDAnalysisTests/rng.py

```diff
@@ -0,0 +1,39 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+import numpy as np
+from numpy.testing import TestCase
+import random
+
+seed = 3856349111 # from: np.uint32(hash("MDAnalysis"))
+
+class RandomState(TestCase):
```

Comment:
I think Max is right, just make a function that does the current setUp
code, then calls random and returns that

On Sat, 27 Aug 2016, 1:05 p.m. mnmelo, <notifications@github.com> wrote:

> In testsuite/MDAnalysisTests/rng.py
> <https://github.com/MDAnalysis/mdanalysis/pull/959#discussion_r76515152>:
>
> > +# Released under the GNU Public Licence, v2 or any higher version
> > +#
> > +# Please cite your use of MDAnalysis in published work:
> > +#
> > +# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
> > +# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
> > +# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
> > +#
> > +
> > +import numpy as np
> > +from numpy.testing import TestCase
> > +import random
> > +
> > +seed = 3856349111 # from: np.uint32(hash("MDAnalysis"))
> > +
> > +class RandomState(TestCase):
>
> Or, in slightly hackish way, MDAnalysisTests.rng could store the
> np.random.RandomState objects in a module-level dict hashed by the test's
> class instance. This would emulate class-based caching, without the need
> for explicitly inheriting anything.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/959/files/54aea15ea6abcd6149203af2346bd98eb7312a18#r76515152>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI0jB1G4cPKypgCGsoqS6B-yX81kVK2kks5qkCgcgaJpZM4JuZ8R>
> .
>